spidermon/contrib/scrapy/pipelines.py

```diff
@@ -3,19 +3,28 @@
 import json
 from io import BytesIO
 from collections import defaultdict
+try:
+    from collections.abc import Mapping
 
+except ImportError:
+    # Backward compatiblity
+    from collections import Mapping
 from scrapy.exceptions import DropItem, NotConfigured
 from scrapy.utils.misc import load_object
 from scrapy.exporters import JsonLinesItemExporter
 from scrapy import Field, Item
 from scrapy.utils.python import to_native_str
 
-from spidermon.contrib.validation import SchematicsValidator, JSONSchemaValidator
+from spidermon.contrib.validation import (
+    SchematicsValidator,
+    JSONSchemaValidator,
+    CerberusValidator,
+)
 from spidermon.contrib.validation.jsonschema.tools import get_schema_from
 from schematics.models import Model
 
 from .stats import ValidationStatsManager
-
+# import ipdb
```

Comment:
Remove comments

Reply:
Yeah, just checked. Do apologize for missing this one.